powerpc/README.md

```diff
@@ -25,38 +25,64 @@ You need these dependencies, they can be provided by OS package system (e.g. apt
 - libcurl-devel
 - maven
 - lsf-tools 2.0
+- hwloc
+- gdrcopy
+
+If using lmod and your system has the necessary modules you can use:
+```shell
+module load gcc/7.4.0
+module load python/3.7.0
+module load cmake/3.17.3
+module load boost/1.66.0
+module load cuda/10.1.243
+module load zlib
+module load texinfo
+module load openblas
+module load netlib-lapack
+module load hwloc
+module load gdrcopy
+```
+
 ### Setup the environment
 Using regular python you just need to make sure that you have an environment.
-If you don't have an environment you can create it with:
+NOTE: if using Lmod, make sure you have loaded your python packages before creating or starting your python environment.
+First lets define an environment variable to keep track of the environment path:
 ```shell
-python -m venv PATH_TO_YOUR_ENV_PREFIX
+export VIRTUAL_ENV=PATH_TO_YOUR_ENV_PREFIX
+mkdir $VIRTUAL_ENV
 ```
-Before run any command first you need to activate your environment with:
+If you don't have an environment you can create it with:
 ```shell
-source PATH_TO_YOUR_ENV_PREFIX/bin/activate
+python -m venv $VIRTUAL_ENV
 ```
-Then install the python dependencies in your environment:
+Before run any command first you need to activate your environment with:
 ```shell
-pip install -r blazingsql/powerpc/requirements.txt
+source $VIRTUAL_ENV/bin/activate
 ```
-Note: All the python dependencies in *requirements.txt* are just simple python packages that doesn't need any gcc/c++ compilation, i.e. they are only pure python packages.
+
+
 ### Build & install BlazingSQL
+It is recommended you setup a build folder and export to the following variable before you begin building:
+`export BLAZINGSQL_POWERPC_TMP_BUILD_DIR=PATH_TO_A_BUILD_FOLDER`
+
 Run the build script and pass your environment folder (prefix path) as argument:
 ```shell
 cd blazingsql
-bash powerpc/build.sh PATH_TO_YOUR_ENV_PREFIX
+bash powerpc/build.sh $VIRTUAL_ENV
```

Comment:
Should it be install.sh?

Reply:
It might be helpful to recommend piping the output to a file, like this
```
bash powerpc/[build.sh|install.sh] $VIRTUAL_ENV | tee out.txt
```

so that it is possible to inspect the file if something goes wrong and, e.g.,  the terminal has closed.